engine/events/print_unown.asm

```diff
@@ -74,7 +74,12 @@ _UnownPrinter:
 	jr nz, .pressed_b
 
 	ldh a, [hJoyPressed]
+	vc_patch print_forbid_1
+if DEF(_CRYSTALVC)
+	and 0
+else
 	and A_BUTTON
+endc
```

Comment:
To get the size of the patch without diffing ROMs:
```suggestion
endc
	vc_patch_end print_forbid_1
```
Likewise for the rest.

Reply:
https://github.com/pret/pokecrystal/pull/882/commits/188faa5462573d47101ae1d0e55919d92237cb80 should resolve this.